--- velvet_0.7.20.orig/Makefile
+++ velvet_0.7.20/Makefile
@@ -5,12 +5,12 @@
 OPT = -O3
 DEF = 
 
-Z_LIB_DIR=third-party/zlib-1.2.3
-Z_LIB_FILES=$(Z_LIB_DIR)/*.o
+# Z_LIB_DIR=third-party/zlib-1.2.3
+# Z_LIB_FILES=$(Z_LIB_DIR)/*.o
 
 # Mac OS users: uncomment the following lines
-# Z_LIB_FILES=
-# LDFLAGS = -lm -lz
+Z_LIB_FILES=
+LDFLAGS = -lm -lz
 # CFLAGS = -Wall -m64
 
 # Sparc/Solaris users: uncomment the following line
@@ -23,13 +23,13 @@
 
 clean :
 	-rm obj/*.o obj/dbg/*.o ./velvet* 
-	cd $(Z_LIB_DIR) && make clean
+	# cd $(Z_LIB_DIR) && make clean
 
 cleanobj: 
 	-rm obj/*.o obj/dbg/*.o 
 
 zlib : 
-	cd $(Z_LIB_DIR); ./configure; make; rm minigzip.o; rm example.o
+	# cd $(Z_LIB_DIR); ./configure; make; rm minigzip.o; rm example.o
 
 velveth : $(OBJ) 
 	$(CC) $(CFLAGS) $(OPT) $(LDFLAGS) -o velveth obj/tightString.o obj/run.o obj/recycleBin.o obj/splay.o obj/splayTable.o obj/readSet.o obj/crc.o $(Z_LIB_FILES)
--- velvet_0.7.20.orig/src/readSet.c
+++ velvet_0.7.20/src/readSet.c
@@ -28,11 +28,7 @@
 #include "tightString.h"
 #include "readSet.h"
 
-#if defined(_WIN32) || defined(__WIN32__) || defined(WIN32)
-#include "../third-party/zlib-1.2.3/Win32/include/zlib.h"
-#else
-#include "../third-party/zlib-1.2.3/zlib.h"
-#endif
+#include "zlib.h"
 
 ReadSet *newReadSet()
 {
